sssom/util.py

```diff
@@ -316,7 +316,7 @@ def filter_redundant_rows(df: pd.DataFrame, ignore_predicate=False) -> pd.DataFr
         key = [SUBJECT_ID, OBJECT_ID, PREDICATE_ID]
     dfmax: pd.DataFrame
     dfmax = df.groupby(key, as_index=False)[CONFIDENCE].apply(max).drop_duplicates()
-    max_conf = {}  # type: Dict[Tuple[Any]]
+    max_conf = {}  # type: Dict[Any, Any]
```

Comment:
why not `max_conf: Dict[Any, Any] = {}`? I think this syntax was added in py36, which is the minimum version for this ontology

Reply:
Yeah, realized that later and made the change. Thanks for checking!

Reply:
yes, you're right either are accepted by MyPy, but it's nicer to use the syntax because then it's actually stored for introspection in python in max_conf.__annotations__